docs/docs/reference/built-in-components/gatsby-head.md

```diff
@@ -88,6 +88,7 @@ You'll need to be aware of these things when using Gatsby Head:
 - The `Head` function needs to return valid JSX.
 - Valid tags inside the `Head` function are: `link`, `meta`, `style`, `title`, `base`, `script`, and `noscript`.
 - Data block `<script>` tags such as `<script type="application/ld+json">` can go in the `Head` function, but dynamic scripts are better loaded with the [Gatsby Script Component](/docs/reference/built-in-components/gatsby-script/) in your pages or components.
+- As of now, you can't access context provided via [`WrapRootElement`](https://www.gatsbyjs.com/docs/reference/config-files/gatsby-browser/#wrapRootElement) inside the `Head` function
```

Comment:
do you mean `apiCallbackContext: object, pluginOptions: pluginOptions` here? should we provide an example maybe or make it more clear?

Reply:
Sure! I'll add an example